src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue33034.cs

```diff
@@ -0,0 +1,32 @@
+#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS // SafeAreaEdges not supported on Catalyst and Windows
```

Comment:
for these can you use #if ANDROID OR IOS? The fails defines are specifically for tests that should be passing on that platform but aren't